src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -961,7 +961,9 @@ private async Task<bool> SendCallStack(SessionId sessionId, ExecutionContext con
                 string function_name = frame["functionName"]?.Value<string>();
                 string url = frame["url"]?.Value<string>();
                 if (!(function_name.StartsWith("wasm-function", StringComparison.Ordinal) ||
-                        url.StartsWith("wasm://wasm/", StringComparison.Ordinal) || function_name == "_mono_wasm_fire_debugger_agent_message"))
+                        url.StartsWith("wasm://wasm/", StringComparison.Ordinal) ||
```

Comment:
the script parsed event has another different condition, should they be the same?

Reply:
done.

Reply:
Maybe convert to a method, so the two places stay in sync? `IsAWasmUrl`?